spec/api/xml_spec.rb

```diff
@@ -79,6 +79,17 @@
           expect(result).to match(/<field name="id">1<\/field>/)
         end
 
+        # add a single hash ("doc")
+        it 'should create an add from a hash formatted for atomic updates' do
+          data = {
+            :id=>1,
```

Comment:
Yep, I mainly want a consistent style. I don't care what style, just pick one and go with it.